ui/app/scripts/task/views/executions.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+	<div class="col-md-12">
+		<div class="col-md-12 table-filter">
+			<div class="col-md-4 col-md-offset-8">
+				<input type="text" class="form-control" ng-model="filterQuery" id="filterTable" placeholder="Quick filter">
+			</div>
+		</div>
+	</div>
+</div>
+<table class="table table-striped table-hover">
+	<thead>
+		<tr>
+			<th>Execution Id</th><th>Task Name</th><th>Start Time</th>
+			<th>End Time</th><th>Exit Code</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr dir-paginate="item in pageable.items | itemsPerPage: pageable.pageSize | filter:filterQuery" total-items="pageable.total"
+		    ng-show="!item.inactive">
+			<td>{{item.executionId}}</td>
+			<td>{{item.taskName}}</td>
+			<td xd-date-time="{{item.startTime}}"></td>
```

Comment:
This `startTime` value seems to have broken when used with `xd-date-time` directive. I see `invalid Date` as the value.